kokoro/config/test/ops_agent/release/centos9.gcl

```diff
@@ -0,0 +1,11 @@
+import 'common.gcl' as common
+import '../../image_lists.gcl' as image_lists
+
+config build = common.ops_agent_test {
+  params {
+    platforms = [
+      'rhel-9',
+      'rocky-linux-9',
+    ]
```

Comment:
This should be:

`platforms = image_lists.centos.distros.rockylinux9.release`

and let's rename this file as well to rockylinux9.

Reply:
Changed in: [8654fa1](https://github.com/GoogleCloudPlatform/ops-agent/pull/812/commits/8654fa10aa977e748769f77935aa2bd9dc07b352)